data/Genrobotics.mdx

```diff
@@ -0,0 +1,16 @@
+---
+name: Genrobotics
+description: Genrobotics, the leading Robotics company in India, primarily focusing on designing and development of Robotic solutions. Our innovation “Bandicoot”- manhole cleaning robot, has been making transformative changes in many sanitation workers who were daring themselves every time without thinking of the extremities hidden in the manholes and ensuring cleanliness to all. We are spearheading the perseverance to create, develop and implement systems of change that are beneficial to all sentient beings and to the environment- by triggering off the distressed elements of the people and fostering a better place to live.
+founded: June 21 2017
+based: Both
+domain: Computer And robotics
+logo: genrobotics.png
+location: Thiruvananthapuram
+address: Genrobotic Innovations Pvt Ltd CDAC Building, Technopark Campus Thiruvananthapuram, Kerala - 695581 India.
+technologies:
```

Comment:
Technologies here should be more specific. For example, If a company works in AI/ML they use specific tech like Python, and Tensorflow. etc. It's also OK if you completely remove the technologies part